johnwayodi/sm-api
# #161524948 fix user endpoint behaviour

#### What does this PR do?
Fix validation glitch on user endpoints 
#### Description of Task to be completed?
1. check correctly for input in api docs
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api.git`
2. Ensure you are in the cloned repo, **sm-api** by default.
3. Create a virtual environment using virtualenv and _pip install -r requirements.txt_
4. `git checkout ch-update-api-docs-161498848`
5. Run application and head over to /apidocs to view dicumentation.
#### Any background context you want to provide?
The request for a single user for GET and DELETE were not working as apidocs is validating wrongly
#### What are the relevant pivotal tracker stories?
[#161524948](https://www.pivotaltracker.com/story/show/161524948)
